ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -504,6 +445,14 @@ else
 	#
 	sh /etc/init.d/rc.logging
 
+	#
+	# Start the rc_input driver.
+	#
+	if [ $USE_IO == no -o $IO_PRESENT == no]
```

Comment:
I get `nsh: [: syntax error` now, adding a space `no ]` fixes it.

Reply:
Sorry I didn't catch that. :/  Corrected and pushed up. Thanks!